### PR TITLE
Refactor array move into util service.

### PR DESCRIPTION
**Pull request by jmuharsky**
Moved container move logic into ContainerService.
Moved dashboard initialization to ExplorerService.

### CODE REVIEW

File: client/components/container/container-service.js [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+
+    if (opt_autoSelect) {
+      this.dashboardSvc.selectContainer(container);
+    }
+
+    return container;
+  }
+
+  /**
+   * Removes the specified container from the curernt dashboard.
```
Comment (klausw) [inline client/components/container/container-service.js]: s/curernt/current/

File: compile.sh [hunk 2]
```diff
@@ -15,7 +15,11 @@ Die () {
   "Run this as ./compile.sh in the PerfKitExplorer directory."
 
 echo "* Clean out the existing deployment content."
-rm -r -f deploy
+rm -r -f deploy/client/components
```
Comment (klausw) [inline compile.sh]: Should this be a separate commit? The relation to the main change is unclear.

**Comment by klausw** [comment]
LGTM

File: client/components/container/container-service.js (hunk 1 above)
Comment (jmuharsky) [inline client/components/container/container-service.js]: Done.

File: compile.sh (hunk 2 above)
Comment (jmuharsky) [inline compile.sh]: Yes, this was a merged PR that I got overzealous on the rebasing.
